testdata/p4_16_errors_outputs/issue561-1.p4-stderr

```diff
@@ -7,9 +7,6 @@ issue561-1.p4(29)
 issue561-1.p4(19)
 header_union U {
              ^
-issue561-1.p4(30): [--Werror=type-error] error: Field setValid is not a member of structure header_union U
+issue561-1.p4(30): [--Werror=invalid] error: u.setValid: Invalid setValid and setInvalid are not defined for header unions
```

Comment:
why change this error message?

Reply:
It is due to the new error message added in the Member postoder method.